### PR TITLE
Use a timesheet tag instead of jQuery trickery to build the timesheets list (WIP).

### PR DESCRIPTION
**Pull request by cdubz**
Here is an initial attempt to break out the `timesheets` tag in to individual `timesheet` tags per discussion in #22. This uses a method outlined by [riotjs issue #1826](https://github.com/riot/riot/issues/1826).

### CODE REVIEW

File: core/static/core/tags/timesheet.tag [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        goToEntries(e) {
+            document.location.href = entriesUrl + e.item.id;
+        }
+
+
+        saveTimesheet(e) {
+            e.preventDefault();
+            let tr = self.root;
+            let td = $(tr).find('td');
+            self.name = $(td[0]).find('input').val();
```
Comment (cdubz) [inline core/static/core/tags/timesheet.tag]: This still seems a bit hackish. Is there a better way to do this with riotjs?

Comment (cdubz) [inline core/static/core/tags/timesheet.tag]: This still seems a bit hackish. Is there a better way to do this with riotjs?

Comment (overshard) [inline core/static/core/tags/timesheet.tag]: @cdubz yep, add a "ref" attribute to the input. You can then call it with self.refs.<nameOfRef>.value

**Comment by overshard** [comment]
Great start, thank you very much for this.

**Pull request merged**